src/app/left-bar/left-bar-button/left-bar-button.component.html

```diff
@@ -28,22 +28,6 @@
       </span>
       {{ buttonLabel }}
     </a>
-    <div
```

Comment:
to check

Reply:
It sounds like these are no longer supported - we should dig into this and see if it is possible. toreplace somehow.